packages/app/ui/containers/Spot.js

```diff
@@ -38,7 +39,15 @@ export default function Spot ({ id }) {
               <meta name="description" content={`${spot.city} tide timetables and charts for the next 10 days`} key="description" />
               <meta name="keywords" content={`${spot.city} tide times,${spot.city} high tide,${spot.city} tide chart,${spot.city} tide tables,${spot.city} low tide`} key="keywords" />
             </Head>
-            <Tides spot={spot} />
+            <Container>
```

Comment:
i think we should only have Semantic container on the top level of the app - not sprinkled around the codebase